src/main/java/com/alex/poseidon/controllers/BidListController.java

```diff
@@ -1,55 +1,92 @@
 package com.alex.poseidon.controllers;
 
-import com.alex.poseidon.models.BidList;
+import com.alex.poseidon.interfaces.BidListControllerInterface;
+import com.alex.poseidon.models.BidListModel;
+import com.alex.poseidon.models.UserModel;
+import com.alex.poseidon.repositories.BidListRepository;
+import com.alex.poseidon.services.BidListService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.Pbkdf2PasswordEncoder;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
 import javax.validation.Valid;
 
 
 @Controller
-public class BidListController {
-    // TODO: Inject Bid service
+public class BidListController implements BidListControllerInterface {
 
+    @Autowired
+    BidListService bidListService;
+
+    @Override
```

Comment:
Each method in each controller needs to have JavaDoc for it

Reply:
Definitive JavaDoc implemented for BidListController/HomeController/LoginController
Temporary JavaDoc implemented for CurveController/RatingController/RuleNameController/TradeController, to go through later in each story.